src/main/java/com/qubit/solution/fenixedu/integration/cgd/services/studentPhoto/CgdStudentPhotoClient.java

```diff
@@ -0,0 +1,157 @@
+package com.qubit.solution.fenixedu.integration.cgd.services.studentPhoto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.ws.BindingProvider;
+
+import org.fenixedu.academic.domain.Employee;
+import org.fenixedu.academic.domain.Person;
+import org.fenixedu.academic.domain.student.Student;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Iterables;
+import com.qubit.solution.fenixedu.bennu.webservices.services.client.BennuWebServiceClient;
+
+import services.caixaiu.cgd.wingman.studentPhotoService.IStudentPhotoService;
+import services.caixaiu.cgd.wingman.studentPhotoService.Member;
+import services.caixaiu.cgd.wingman.studentPhotoService.StudentPhoto;
+import services.caixaiu.cgd.wingman.studentPhotoService.StudentPhotoService;
+
+public class CgdStudentPhotoClient extends BennuWebServiceClient<IStudentPhotoService> {
+
+    private static Logger logger = LoggerFactory.getLogger(CgdStudentPhotoClient.class);
+
+    @Override
+    protected BindingProvider getService() {
+        return (BindingProvider) new StudentPhotoService().getBasicHttpBindingPhotoStudentService();
+    }
+
+    public static class MemberCGD {
+        String category;
+        String memberNumber;
+
+        public boolean isStudent() {
+            return "91".equals(category) && memberNumber.length() <= 5 && getMemberAsNumber() != null;
```

Comment:
@pmmsa _magic number_ such as this 91 and 71 or 81 should be made static final in the class.
Example:
```
public static final String STUDENT_IDENTIFICATION="91";
public static final String STAFF_IDENTIFICATION_VAL1 = "71";
public static final String STAFF_IDENTIFICATION_VAL2="81"; 
```

Actually if you know the difference between 71 and 81 and make is explicit in the name would be awesome!